operator/pkg/ciliumenvoyconfig/envoy_config.go

```diff
@@ -129,10 +130,15 @@ func (m *manager) getResources(svc *slim_corev1.Service) ([]ciliumv2.XDSResource
 }
 
 func (m *manager) getClusterResources(svc *slim_corev1.Service) ([]ciliumv2.XDSResource, error) {
+	lbPolicy, ok := envoy_config_cluster_v3.Cluster_LbPolicy_value[strings.ToUpper(m.algorithm)]
+	if !ok {
+		lbPolicy = int32(envoy_config_cluster_v3.Cluster_ROUND_ROBIN)
+	}
```

Comment:
Should the user be warned if they put an unacceptable algorithm and we fallback to round robin?